continuous-integration/linux/platform-dependent/ubuntu-16.04/setup.sh

```diff
@@ -33,5 +33,5 @@ function install_www_deb {
     rm -r $TMP_DIR || exit $?
 }
 
-install_www_deb https://www.zivid.com/hubfs/softwarefiles/releases/1.7.0+a115eaa4-4/u16/zivid-telicam-driver_2.0.0.1-1_amd64.deb || exit $?
-install_www_deb https://www.zivid.com/hubfs/softwarefiles/releases/1.7.0+a115eaa4-4/u16/zivid_1.7.0+a115eaa4-4_amd64.deb || exit $?
+install_www_deb https://www.zivid.com/hubfs/softwarefiles/releases/1.8.0+89ae8b3e-39/u16/zivid-telicam-driver_2.0.0.1-1_amd64.deb || exit $?
+install_www_deb https://www.zivid.com/hubfs/softwarefiles/releases/1.8.0+89ae8b3e-39/u18/zivid_1.8.0+89ae8b3e-39_amd64.deb || exit $?
```

Comment:
This is the wrong link, you have the Ubuntu 18 deb here instead of the Ubuntu 16 one.

Reply:
Good catch.

Reply:
I didn't really catch it, though! :D I actually missed it, and didn't notice until I saw the red CI and did some digging.

Reply:
Can you show me how that digging works? I tried to search for it but couldn't find anything.